Documentation/project-docs/clr-configuration-knobs.md

```diff
@@ -79,7 +79,7 @@ Name | Description | Type | Class | Default Value | Flags
 
 Name | Description | Type | Class | Default Value | Flags 
 -----|-------------|------|-------|---------------|-------
-`ARMEnabled` | Set it to 1 to enable ARM | `DWORD` | `UNSUPPORTED` | `(DWORD)0` | 
+`ARMEnabled` | AppDomain Resource Monitoring. Set to 1 to enable it | `DWORD` | `UNSUPPORTED` | `(DWORD)0` |
 
 #### Assembly Loader Configuration Knobs
```

Comment:
Changes should be added to `clr-configuration-knobs.csx` file and than changed script could be used to generate `clr-configuration-knobs.md` file. Otherwise, changes made directly in `.md` file will be overwritten next time `clr-configuration-knobs.csx` will be run.

However, in this case, due to the presence of knob desription in `clrconfigvalues.h` it could be enough to rerun `clr-configuration-knobs.csx` and check if script picked changes from headers.